src/index.ts

```diff
@@ -17,11 +17,21 @@ interface CustomContainerOptions {
    * @defaultValue "div"
    */
   containerTag?: string;
+  /**
+   * @defaultValue {className: string[]}
+   */
+  titleElement?: Record<string, unknown> | null;
+  /**
+   * @defaultValue undefined
+   */
+  additionalProperties?: (className: string, title: string) => Record<string, unknown>;
 }
 
 const DEFAULT_SETTINGS: CustomContainerOptions = {
   className: "remark-container",
   containerTag: "div",
+  titleElement: {},
+  additionalProperties: undefined,
```

Comment:
IMHO we can use `{}` for its default value for convenience.

Reply:
Do you mean the {} for the titleElement or the additionalProperties ?

Reply:
I see it makes sense.